packages/ember-metal/lib/property_get.js

```diff
@@ -76,10 +76,10 @@ export function get(obj, keyName) {
       if (meta && meta.peekWatching(keyName) > 0) {
         ret = meta.peekValues(keyName);
       } else {
-        ret = obj[keyName];
+        ret = possibleDesc;
```

Comment:
var name should be different, as in this position (and bellow) it actually cannot be a descriptor any-longer.